index.js

```diff
@@ -16,11 +16,11 @@ module.exports = function (content) {
       children = result.attributes.children,
       componentProps = renderProps(props);
 
-    let imports = '';
+    let imports = 'import React from "react";';
```

Comment:
You need to check if React is already defined before importing it, otherwise it may create some issues